librad/src/git/storage/fetcher.rs

```diff
@@ -262,6 +262,20 @@ pub mod error {
         #[error(transparent)]
         Pool(#[from] super::PoolError),
     }
+
+    #[derive(Debug, Error)]
+    pub enum FetchError {
+        #[error("Fetch limit {limit} exceeded. Amount fetched: {amount_fetched} from {remote}")]
```

Comment:
We try and follow these rules for displaying errors https://github.com/radicle-dev/radicle-decisions/blob/master/proposals/0002.md#the-behavior-of-display
```suggestion
        #[error("fetch limit {limit} exceeded, fetched {amount_fetched} from {remote}")]
```

Reply:
Fixed